app/src/main/java/org/oppia/android/app/onboarding/AudioLanguageFragmentPresenter.kt

```diff
@@ -158,6 +159,30 @@ class AudioLanguageFragmentPresenter @Inject constructor(
       }
   }
 
+  private fun loginToProfile(profileId: ProfileId) {
```

Comment:
```suggestion
  private fun logInToProfile(profileId: ProfileId) {
```

Nit: should be verb form.

Reply:
Refactored as suggested.